main.cpp

```diff
@@ -65,7 +86,22 @@ struct Wrapper
  send me a DM to review your pull request when the project is ready for review.
 
  Wait for my code review.
- */
+ */
+
+template<typename T>
+void variadicHelper(T first)
+{
+    Wrapper<T> w(std::move(first));
+    w.print();
+}
```

Comment:
Project9.1 challenge 2:
There is another recursive solution.  it is possible for `Args ...` to be empty.
your task: remove your implementation for instruction `4)` and implement this alternate solution.
implement it **after** your existing `variadicHelper` function.
hint: what does the **recursive call** actually look like when called with an empty Variadic Parameter list?

- [ ] revise